components/browser/state/src/main/java/mozilla/components/browser/state/state/MediaSessionState.kt

```diff
@@ -0,0 +1,37 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.browser.state.state
+
+import mozilla.components.concept.engine.mediasession.MediaSession
+import java.util.UUID
+
+/**
+ * Value type representing a media session on a website.
+ *
+ * @property id Unique ID for this media session.
+ * @property sessionId Unique Session ID for this media session.
+ * @property isActive True if media session is active.
+ * @property controller The [MediaSession.Controller] for controlling playback of this media session.
+ * @property metadata The [MediaSession.Metadata] for this media session.
+ * @property elementMetadata The [MediaSession.ElementMetadata] for this media session.
+ * @property playbackState The current simplified [MediaSession.PlaybackState] of this media session.
+ * @property features The [MediaSession.Feature] for this media session.
+ * @property positionState The current simplified [MediaSession.PositionState] of this media session.
+ * @property muted True if media session is muted.
+ * @property fullscreen True if media session is fullscreen.
+ */
+data class MediaSessionState(
+    val id: String = UUID.randomUUID().toString(),
+    val sessionId: String? = null,
```

Comment:
As mentioned above, I think this is now unnecessary since we attach the state to a session.

Reply:
Agreed, I'll remove